### PR TITLE
profiles/arch/powerpc: mask flags for unkeyworded depends

### PR DESCRIPTION
**Pull request by dracwyrm**
This needs to be merged before my other PRs.

Packages being added to the main repo that are not arm and powerpc
keyworded.

Signed-off by: Jonathan Scruggs <j.scruggs@gmail.com>

@SoapGentoo

### CODE REVIEW

File: profiles/arch/arm/use.mask [hunk 1]
```diff
@@ -5,6 +5,13 @@
 -cpu_flags_arm_iwmmxt
 -iwmmxt
 
+# Jonathan Scruggs <j.scruggs@gmail.com> (10 Aug 2017)
```
Comment (mgorny) [inline profiles/arch/arm/use.mask]: I honestly doubt you're going to get all those flags global, so they belong in `package.use`.

Comment (dracwyrm) [reply profiles/arch/arm/use.mask]: Oh good catch. I opened use.mask instead of package. Oops. I did mean package mask. :)

Comment (dracwyrm) [reply profiles/arch/arm/use.mask]: D'oh. Did it for all three.

File: profiles/arch/arm/package.use.mask [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+# Jonathan Scruggs <j.scruggs@gmail.com> (10 Aug 2017)
+# dependencies not keyworded yet
+field3d
+osl
+partio
+ptex
```
Comment (SoapGentoo) [inline profiles/arch/arm/package.use.mask]: these need to be package specific (see the other non-commented lines)

Comment (dracwyrm) [reply profiles/arch/arm/package.use.mask]: Having a cold really sucks. XD

**Pull request closed without merging**